### PR TITLE
Update project profile 3773

### LINKED ISSUE #3773: Update Project Profile: Home Unite Us
### Overview
As a developer, we must ensure our project's details are up to date on the HackforLA website. We need to update the [Home Unite Us](https://www.hackforla.org/projects/home-unite-us) page's leadership.

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of [_projects/home-unite-us.md](https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md) and find the `leadership` variable. 
- [x] Remove the following persons from the `leadership` variable
    - [x] Tracy Michelle
    - [x] Regan Hutson
    - [x] Kristy Points
    - [x] Ben Ross
- [x] Add the following persons `name`, `role`, `links`, and `picture` to the `leadership` variable
    - [x] Name: Jed Stewart
GitHub Handle: jed-stewart
Role: Developer
Slack Member ID: U04A3J6V0HY
    - [x] Name: Mira Kinebuchi
GitHub Handle: mira-kine
Role: Developer
Slack Member ID: U0411CAL13N
    - [x] Name: Stephen Barkley-Yeung
GitHub Handle: stevbark
Role: Developer
Slack Member ID: U041403GHEC
    - [x] Name: Kristelle Gumaru
GitHub Handle: K-Gumarama
Role: Product Manager
Slack Member ID: U031VJ99ATU
    - [x] Name: Sanya Nijhawan
GitHub Handle: sanya301
Role: Product Manager
Slack Member ID: U031FLYS561
    - [x] Name: Raji Pradheap
GitHub Handle: rpradheap
Role: Product Manager
Slack Member ID: U02R9MKA6KH
    - [x] Name: Bryce Lednar
GitHub Handle: brycelednar
Role: Product Manager
Slack Member ID: U049TSDG2SX
    - [x] Name: Farah Khan
GitHub Handle: fourmatte
Role: UX/UI Designer
Slack Member ID: U01JC8VEKMH
    - [x] Name: Eduardo De La Rosa
GitHub Handle: edela0015
Role: UX/UI Designer
Slack Member ID: U0430G2T4AC
    - [x] Name: Flora Osmond
GitHub Handle: itzflowa
Role: UX/UI Designer
Slack Member ID: U047ZT4GP5E

- [x] Test the Home Unite Us Project Page at [http://localhost:4000/projects/home-unite-us](http://localhost:4000/projects/home-unite-us) on your local development environment to ensure the changes are being properly displayed

### Instructions
- Be sure to use valid YAML syntax and match the existing structure of [_projects/home-unite-us.md](https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md)
- For the slack link, append the team member's slack ID to https://hackforla.slack.com/team/
- For the github link, append the team member's github handle to https://github.com/
- For the picture URL, append the team member's github handle to https://avatars.githubusercontent.com/

### Resources
- [Home Unite Us project page](https://www.hackforla.org/projects/home-unite-us)
- [_projects/home-unite-us.md](https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

### PR DESCRIPTION
**Pull request by Satenik-Ba**
Fixes #3773

### What changes did you make and why did you make them ?

  - In the Current Project Team section, updated the role of team members currently listed as "Project Manager" to "Product Manager"

<details>
<summary>Visuals before changes are applied</summary>

Screen Shot 2023-02-14 at 5 56 50 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screen Shot 2023-02-14 at 5 59 16 PM

</details>

### CODE REVIEW

**Comment by vanessavun** [comment]
Availability: 12PM-2PM Wed 2/15, evening Sat 2/18
ETA: Wed 2/15 by 2PM

File: _projects/home-unite-us.md [hunk 1]
```diff
@@ -37,25 +37,26 @@ leadership:
       github: "https://github.com/erikguntner"
     picture: https://avatars.githubusercontent.com/erikguntner
   - name: Kristelle Gumaru
-    role: Project Manager
+    role: Product Manager
+
```
Comment (vanessavun) [inline _projects/home-unite-us.md]: Great job changing all associated "Project" to "Product" as requested in the fix and visually it reflects the changes. However, it looks like you created an extra line here. Please remove the extra line for consistency.

**Review by vanessavun — CHANGES_REQUESTED** [review]
Thanks for working on this issue as requested. Visually it checks out and your commits are into gh-pages from your username and branch. Please review my comment about the extra line in line 41 and resubmit. 
Edit:
I also notice there are commit requests to merge the 'gh-pages' branch into your branch. I suggest doing a git pull of the latest changes from gh-pages and submitting 1 commit to the changes you made for the fix.

**Comment by blulady** [comment]
@christinaor Please add your eta & availability for this PR review when you get a chance.

**Comment by christinaor** [comment]
Availability: Thurs 4-5pm
ETA: Thurs 5pm

**Comment by christinaor** [comment]
The changes look good on Docker! I agree with Vanessa on removing line 41 to be consistent with the rest of the formatting.

I also noticed multiple commits here. If your fork of the website was synced, and then it was merged into your local gh-pages and subsequently the update-project-profile-3773 branch, I think it is fine this time as there are multiple ways to sync your branch to the upstream gh-pages.

I will review after the resubmit - great job so far!

**Review by christinaor — APPROVED** [review]
Confirmed the change from "Project Manager" to "Product Manager" are still present when checking the website via Docker, and the empty line after line 40 has been removed. Great job, Satenik!

File: _projects/home-unite-us.md (hunk 1 above)
Comment (Satenik-Ba) [reply _projects/home-unite-us.md]: Hi @vanessavun changes made, but wasnt able to request a review again. if you can take a look over the changes will be great. thank you!

**Review by vanessavun — APPROVED** [review]
Thanks for making the changes! All in the clear.

**Comment by blulady** [comment]
@vanessavun & @christinaor were you able to pull the branch into your local repo? 
`git checkout -b satenik-ba-update-project-profile-3773`
`git pull https://github.com/Satenik-Ba/website update-project-profile-3773`
I am getting a merge conflict when I pull the branch into my local. Looks like line 39, 40 and 47 need to be deleted...
Getting a merge conflict warning when I try to merge this branch into the code base. Am not seeing it in the github files changed.... Please pull into your local repos and let me know if this is a me thing...

**Comment by christinaor** [comment]
@blulady The pull is now asking me to merge lines 39 to 44, which are from the addition of Kristelle Gumaru. I synced my gh-pages branch before pulling Satenik's branch again, so I think the merge conflict might be from the recently merged solutions. I can still see the changes from "Project Manager" to "Product Manager" for lines 40, 46, 52, and 58, but line 40 needs to be accepted through the addition of lines 39 to 44 in the merge conflict, at least from what I can see. Hope this helps!

**Comment by vanessavun** [comment]
@blulady I'll be available tonight to check! ETA 5-6PM.

**Review by vanessavun — APPROVED** [review]
> @vanessavun & @christinaor were you able to pull the branch into your local repo? `git checkout -b satenik-ba-update-project-profile-3773` `git pull https://github.com/Satenik-Ba/website update-project-profile-3773` I am getting a merge conflict when I pull the branch into my local. Looks like line 39, 40 and 47 need to be deleted... Getting a merge conflict warning when I try to merge this branch into the code base. Am not seeing it in the github files changed.... Please pull into your local repos and let me know if this is a me thing...

I'm not getting any merge conflicts but I did confirm that "Project Manager" are all "Product Manager" for lines 40, 45, 52, and 58.

**Comment by vanessavun** [comment]
@christinaor We need your review too.

**Review by christinaor — APPROVED** [review]
Based on what I saw, I think this can be approved with the thought the merge team will take care of the syncing issues between the branches.

**Pull request merged**